metabase/metabase
# [Dashboard] Make parameters widget container not sticky on small devices

Closes https://github.com/metabase/metabase/issues/33670

### Description

Make parameters widget not sticky on small devices. 
Accordingly with existing logic, even mobile can have a sticky header if there is 5 or less filters on the dashboard.

### How to verify
- add 7 filters to the dashboard
- open it on mobile (or simulate on dev tools)
- you should be able to scroll to the questions

### Demo

https://www.loom.com/share/77b1df6fdb624d6eb715acdd4ba61823

### Checklist
